templates/zerver/help/change-a-users-name.md

```diff
@@ -0,0 +1,29 @@
+# Change a user's name
+An administrator can change the name of another user.
+Ordinary users can change their own names,
+so administrators should generally try to teach users how to do this for themselves,
+but administrators can do it when users are unavailable.
+
+Follow the following steps to change the name of a user using an administrator account:
+* Click on the cog ![cog](/static/images/help/cog.png) icon in the upper right corner
```

Comment:
Add parentheses around the icon like this `cog (![cog](/static/images/help/cog.png))`